sympy/stats/crv_types.py

```diff
@@ -484,7 +484,7 @@ def BetaNoncentral(name, alpha, beta, lamda):
 
     Compute cdf with specific 'x', 'alpha', 'beta' and 'lamda' values as follows :
     >>> cdf(BetaNoncentral("x", 1, 1, 1), evaluate=False)(2).doit()
-    exp(-1/2)*Integral(Sum(2**(-_k)*_x**_k/(beta(_k + 1, 1)*factorial(_k)), (_k, 0, oo)), (_x, 0, 2))
+    2*exp(1/2)
```

Comment:
Seems like the possibility to explicitly rewrite `beta` to `gamma` had some positive impact on this.

Reply:
Maybe this:
https://github.com/sympy/sympy/blob/0fbe6d0bff94247c2d12259319387cf5634ea18a/sympy/integrals/transforms.py#L718

Reply:
Probably. Seems like an added bonus! :-)

Reply:
Didn't find any issues that could benefit from it though.